portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,25 @@
+body {
+  background: #202020;
+  color: white;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
+#intro {
+  border: 1px solid black;
+}
+
+img {
+  width: 350px;
+  height: 400px;
+  border: 1px solid white;
+  box-shadow: 0 19px 30px rgba(0,0,0,0.30), 0 15px 12px rgba(0,0,0,0.22);
```

Comment:
Can you [omit the leading zeros](https://google.github.io/styleguide/htmlcssguide.html#Leading_0s) in the rgba values?

Reply:
Found an alternative! Thanks for pointing that out!